public/app/features/alerting/unified/components/settings/VersionManager.tsx

```diff
@@ -296,29 +296,4 @@ function normalizeConfig(config: AlertManagerCortexConfig) {
   return omit(config, ['id', 'last_applied']);
 }
 
-function computeConfigDiff(json1: AlertManagerCortexConfig, json2: AlertManagerCortexConfig): Diff {
```

Comment:
we created a generic `computeVersionDiff` based on this one, to reuse this functionality.